### DISCUSSION TITLE
How do I render with RENDER_ECS?

### DISCUSSION THREAD

**Discussion by hoylu-royberg** [klogging/klogging#254] [Q&A] [answered]
We are in the process of migrating from Logback to Klogging, and we keep our log data in Elastic, parsed with Fluentd. So I wanted to render the log statements to `STDOUT`, using `RENDER_ECS`, but when I tried that, I get the error _"File Configuration : Failed to load class RENDER_ECS"_ during startup. Looking at the source code, I see that `RENDER_ECS` is not included in the standard built-in configuration, so how do I use it? I tried using full namespace as well, but that didn't work either.

Awesome work so far! 🙌

**Comment by hoylu-royberg** [comment]
Forgot to mention that I'm doing the configuration via JSON file:
```JSON
{
  "sinks": {
    "stdout": {
      "renderWith": "RENDER_ECS",
      "sendTo": "STDOUT"
    }
  },
  "logging": [
    {
      "levelRanges": [
        {
          "fromMinLevel": "INFO",
          "toSinks": [
            "stdout"
          ]
        }
      ]
    }
  ]
}

```

**Comment by mjstrasser** [comment, marked as answer]
Hey @hoylu-royberg thanks for using Klogging!

I forgot to add the name `RENDER_ECS` to the list of known renderers in the code. I have fixed in version 0.5.12 that and it should now work.

Please let me know if it renders as you expect (I have not set up and used ElasticSearch myself) or if I need to change it in some way.

Sorry for the inconvenience!

**Reply by hoylu-royberg** [reply]
Wow, thank you for the quick fix and response! 🙌
